### PR TITLE
feature: remove the secrets' columns from the database

### PR DESCRIPTION
**Pull request by MikelAlejoBR**
After migrating all the secrets to sources, we decided that it was time to remove both the database columns for the secrets and the feature flag, since storing secrets in Sources is going to be enabled by default.

## Depends on
* https://github.com/RedHatInsights/notifications-backend/pull/2350 

## Jira ticket
[[RHCLOUD-29443]](https://issues.redhat.com/browse/RHCLOIUD-29443)

### CODE REVIEW

File: database/src/main/resources/db/migration/V1.92.0__RHCLOUD-29443_remove_secrets_columns.sql
Comment (gwenneg) [inline database/src/main/resources/db/migration/V1.92.0__RHCLOUD-29443_remove_secrets_columns.sql:1]: This file needs to be renamed to `V.1.94...`.

Comment (gwenneg) [reply database/src/main/resources/db/migration/V1.92.0__RHCLOUD-29443_remove_secrets_columns.sql:1]: The PR probably needs a rebase as well.

Comment (MikelAlejoBR) [reply database/src/main/resources/db/migration/V1.92.0__RHCLOUD-29443_remove_secrets_columns.sql:1]: Yes, done!

File: database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "basic_authentication",
+DROP COLUMN
+    "secret_token";
+
+ALTER TABLE
+    "endpoint_webhooks"
+DROP COLUMN
+    "basic_authentication",
+DROP COLUMN
+    "secret_token";
```
Comment (gwenneg) [inline database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql:13]: We can also drop the "bearer token" column.

Comment (gwenneg) [reply database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql:13]: Or maybe not, `WebhookProperties#bearerAuthentication` is still on `master`.

Comment (gwenneg) [reply database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql:13]: Or maybe not, `WebhookProperties#bearerAuthentication` is not marked as `@Transient` on master.

Comment (gwenneg) [reply database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql:13]: Sounds good 👍

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**